.gitignore

```diff
@@ -26,3 +26,6 @@ config/config.yml
 config/database.yml
 
 .idea/*
+
+# Ignore macOS metadata
+.DS_Store
```

Comment:
This isn't actually necessary; I just found out I can create a global `.gitignore` file.  We can remove this, or we can leave it in.